### ISSUE TITLE
SubtractFeeFromAmount Inconsistent Behaviour

### ISSUE THREAD

**Issue by VoxR4710** [bitcoin/bitcoin#16108]
Running bitcoind and bitcoin-cli with JSON RPC, 0.18.0 on testnet.
I still have some issue with SubtractFeeFromAmount. It makes no sense at all, though. Stay with me on this for a moment , please:

Balance: 0.00008000
Received: 0.00118410
Balance: 0.00126410
Sent: 0.00118292
Balance: 0.00008000
Sent: 0.00001000
Balance: 0.00007000
Received: 0.00118410
**Balance: 0.00125410
Then to make sure it was nothing about PHP or jsonRPC, I use bitcoin-cli directly:
bitcoin-cli sendtoaddress n3qpExEQbF8orkqY7cpmJu6iUhNVHHcnk3 0.00118292 "" "" true
50458050f042460ea7199a0d0327edbae153e4e0083cc5862ab64322b31819e9
Balance: 0.00007000 (!)**

How is it possible? Sometimes it works, sometimes it doesn't work? For a long time I thought I had written the parameters wrong for json-RPC, but unless the above command is also wrong, it's not me?
All of the above was of course performed with subtractfeefromamount true.
Basically all config values are default or unset, no specific txfees or anything like that.

Thank you so much in advance for any help with this as it's had me stumped for days now.
I made a new issue for clarity (different actual issue) and because I don't know if closed are ever reviewed again.

**Comment by MarcoFalke** [comment]
Duplicate of #16100
